### DISCUSSION TITLE
PK-SIM permeability-SA products etc?

### DISCUSSION THREAD

**Discussion by jmgallo** [Open-Systems-Pharmacology/Forum#780] [Questions & Problems] [answered]
Hi,
In the section, "How are permeability surface-area (PxSA) products predicted in PK-Sim®?" there is a thorough discussion of how PxSA are calculated, but I can't find the actual values used for each organ in a simulation. The actual values used in the ODEs and simulation for each organ are desired. Please advise.
Thanks, jim

**Comment by Aedginto** [comment, marked as answer]
Hi Jim, 
Surface areas can be found in the Anatomy section of the INDIVIDUAL (you must look under ADVANCED and not the default, which is SIMPLE view). P is cellular permeability and the same for all organs. 
Take care,
Andrea

**Comment by RobinM92** [comment]
Hi, 
perhaps to add a (naive) question to this. Which SA would be the one that typically matters for the standard, hepatically metabolized small molecule case? As I would assume CYPs to be found intracellular I would say the interstitial<->intracellular area (e.g. 4.96e+7 cm2 for the liver). Say that we just want to compare P*SA with Q to get a ballpark idea whether our compound will behave closer to well-stirred or to permeability limited behavior, would this be the correct way to go? Or are all P*SA products used in the ultimate CL calculation? 
Thanks a lot!
Cheers,
Robin
